presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -866,6 +866,22 @@ public void testLimitPushDown()
 
         assertEquals(actual.getMaterializedRows().size(), 10);
         assertContains(all, actual);
+
```

Comment:
General comment. I have some problem with test naming. It claims that pushdown is being tested here. While we do not really have any pushdown related assertions. We are just checking if LIMIT queries are not messed up by connector implementation.

Reply:
pre-existing, i can change it still, preferrably as a followup

Reply:
Sure. Thanks